ledger/test-common/BUILD.bazel

```diff
@@ -89,21 +107,51 @@ da_scala_library(
             ["@maven//:org_scala_lang_modules_scala_collection_compat"],
         ),
         ("performance", []),
+        ("package_management", []),
     ]
-    for (target, target_suffix) in test_common_configurations
+    for (target, target_suffix) in test_common_lf_targets.items()
 ]
 
 [
     [
+        genrule(
+            name = "dar-files-lookup-%s.scala" % target_suffix,
+            outs = ["Dars-%s.scala" % target_suffix],
+            cmd = """
+cat > $@ <<EOF
+package com.daml.ledger.test_common
```

Comment:
Please follow the [Java package naming conventions](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names), i.e., don't use underscores. If you don't want to end up with `testcommon`, then please create a package `com.daml.ledger.test.common` or similar.